<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Flashcarder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="Tabs">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="AddFlashcardTab">
      <property name="accessibleName">
       <string/>
      </property>
      <attribute name="title">
       <string>Добавить карточку</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="AddFlashcardLayout">
         <item row="2" column="1">
          <widget class="QPlainTextEdit" name="WordEdit"/>
         </item>
         <item row="4" column="1">
          <widget class="QPlainTextEdit" name="ExamplesEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="WordLabel">
           <property name="text">
            <string>Слово</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="MeaningLabel">
           <property name="text">
            <string>Значение</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPlainTextEdit" name="MeaningEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="ExamplesLabel">
           <property name="text">
            <string>Примечания</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="AddFlashcardButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TestTab">
      <attribute name="title">
       <string>Тест</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QStackedWidget" name="TestPages">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="TestLayout">
          <layout class="QGridLayout" name="gridLayout_10">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QStackedWidget" name="TestQuestion">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="MeaningToWordDoYouRemember">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QLabel" name="MeaningToWordDoYouRememberLabel">
                 <property name="text">
                  <string>Помнишь слово?</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="WordToMeaningDoYouRemember">
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="QLabel" name="WordToMeaningDoYouRememberLabel">
                 <property name="text">
                  <string>Помнишь значение?</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="DidYouRemember">
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="0" column="0">
                <widget class="QLabel" name="DidYouRememberLabel">
                 <property name="text">
                  <string>Идеально вспомнил?</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="TryToRemember">
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QLabel" name="TryToRememberLabel">
                 <property name="text">
                  <string>Запомни</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="TestTabLayout">
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="1" column="1" rowspan="3">
              <widget class="QProgressBar" name="TestProgressBar">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPlainTextEdit" name="TestInformation">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="plainText">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QStackedWidget" name="TestButtons">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="TestYesNoLayout">
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="2">
                  <widget class="QPushButton" name="TestNoButton">
                   <property name="text">
                    <string>Нет</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="TestYesButton">
                   <property name="text">
                    <string>Да</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="TestOkLayout">
                <layout class="QGridLayout" name="gridLayout_9">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="TestOkButton">
                   <property name="text">
                    <string>Ладно</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="TestStartLayout">
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="1" column="0">
            <widget class="QRadioButton" name="QuickTestRadioButton">
             <property name="text">
              <string>По неизученным</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="Line" name="VerticalLine">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QRadioButton" name="WordToMeaningRadioButton">
             <property name="text">
              <string>Слова → значения</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
           <item row="5" column="0">
            <spacer name="VerticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="0">
            <widget class="QPushButton" name="StartButton">
             <property name="text">
              <string>Начать</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="FullTestRadioButton">
             <property name="text">
              <string>По всем</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QRadioButton" name="MeaningToWordRadioButton">
             <property name="text">
              <string>Значения → слова</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="FlashcardsListTab">
      <attribute name="title">
       <string>Все слова</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <widget class="QTableWidget" name="FlashcardsList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Слово</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Значение</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Примечания</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
